OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,97 @@
+        $ docker run -d -e WEBLOGIC_HOST=i<host> WEBLOGIC_PORT=<port> -p 80:80 12213-apache
+
+Run an Apache container to access and load balance to a list of managed servers running standalone 
+ 
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+Run an Apache image to proxy and load balance to a list of managed servers in a cluster
+     
+        Use a list of hosts and ports.
+
+        $ docker run -d -e WEBLOGIC_CLUSTER=host1:port,host2:port,host3:port --net=<some net> -p 80:80 12213-apache
+
+        Or use a cluster URL if it is available
+
+        $ docker run -d -e WEBLOGIC_CLUSTER=<cluster-url> --net=<some net> -p 80:80 12213-apache
+
+The values of **WEBLOGIC_CLUSTER** must be valid, existing containers running WebLogic servers.
+
+If you are using multihost network, remove `--link` and set `--net=<your net>`.
+
+### Admin Server Only Example
+ 
+First make sure you have the WebLogic Server 12.2.1.3 sample and build your own image 12213-domain at [https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-domain].
+
+To start a containerized Admin Server, run:
+
+        $ docker run -d --name wlsadmin -h wlsadmin -p 7001:7001 12213-domain
+
+Start an Apache container by calling:
+
+        $ docker run -d --name apache -e WEBLOGIC_HOST=<admin-host> -e WEBLOGIC_PORT=7001 -p 80:80 12213-apache
+
+Now you can access the WebLogic Admin Console under **http://localhost/console** (default to port 80) instead of using port 7001. You can access the console from a remote machine using the weblgoic admin server's `<admin-host>` instead of `localhost`.
+
+## Provide Your Own Apache Plugin Configuration
+If you want to start the Apache container with some pre-specified `mod_weblogic` configuration:
+
+* Create a `custom_mod_wl_apache.conf` file by referring to `custom_mod_wl_apache.conf.sample` and Chapter 3 @ Fusion Middleware Using Oracle WebLogic Server Proxy Plug-Ins documentation. [https://docs.oracle.com/middleware/12213/webtier/develop-plugin/apache.htm#GUID-231FB5FD-8D0A-492A-BBFD-DC12A31BF2DE]
+
+* Place the `custom_mod_wl_apache.conf` file in a directory on the host machine (for example,`/scratch/apache-config`) and then mount this directory into the container at the location `/config`. By doing so, the contents of host directory `/scratch/apache-config` (and hence `custom_mod_wl_apache.conf`) will become available in the container at the mount point.
+
+This mounting can be done by using the -v option with the `docker run` command as shown below. 
+
+        $ docker run -v /scratch/apache-config:/config -w /config -d -e WEBLOGIC_HOST=<admin-host> -e WEBLOGIC_PORT=7001 -p 80:80 12213-apache
+
+Note: you can also mount the file directly as follows.
+
+        $ docker run -v /scratch/apache-config/custom_mod_wl_apache.conf:/config/custom_mod_wl_apache.conf -w /config -d -e WEBLOGIC_HOST=<admin-host> -e WEBLOGIC_PORT=7001 -p 80:80 12213-apache
```

Comment:
replace /scratch/apache-config/ for host directory  a user when they run this command will not have a .scratch directory